packages/documentation/src/pages/forms/form-tutorial-basic.mdx

```diff
@@ -34,7 +35,13 @@ You can answer the questions with the [documentation](https://github.com/departm
 - What's the benefit description for this form? `new form benefits`
 - Which form template would you like to start with? `Choose 'BLANK: A form without any fields'`
 
-After you run the generator, run `yarn watch` (or restart it if you already had it running) and navigate to http://localhost:3001/new-form. You should see something like this:
+After you run the generator, run
```

Comment:
if we're giving explicit steps we don't have to give "after" instructions. they're kind of confusing because they suggest I should look at the generator document and run something else. 

you could add a note after the first set of steps like 
_After answering the questions, the generator will create several source files for a new form application in `/src/applications/new-form`._

Reply:
Maybe add 
## Run the form application